pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -257,6 +258,18 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, nil
 	}
 
+	// Only allow live migration in managed clusters excluding hypershift clusters
+	isNonHypershiftManagedCluster, err := util.IsNonHypershiftManagedCluster(ctx, r.client)
```

Comment:
I wasnt a fan of this exclusive naming but I couldnt think of a better one. I think the comment above it helps explain that its the set of all managed cluster excluding hypershift.